scrape.js

```diff
@@ -112,6 +121,12 @@ const fetchProfileLinks = async (page, pagesToVisit = 2) => {
   return profileLinks;
 };
 
+/**
+ * Get all activity of the profile
+ * @param {Promise} page Promise of Browser page
+ * @param {Array} profileLinks
+ * @param {String[]} waitUntilOptions
```

Comment:
`profileLinks` array is an array of strings(array of strings can be specified using the jsodcs) and `waitUntilOptions` would have the same type. It would be great to have descriptions for both as well. 
`profileLinks` array stores scraped profile links from all the pages.
`waitUntilOptions` stores puppeteer options.